docs/3.0/cache.md

```diff
@@ -39,6 +39,49 @@ config.cache_store = ActiveSupport::Cache.lookup_store(:solid_cache_store)
 Our computed system do not use MemoryStore in production because it will not be shared between multiple processes (when using Puma).
 :::
 
+## Problem: cached rows on table do not update when record's associations suffer changes
```

Comment:
- remove the `problem`
- we don't need info
- structure can be:
  - `##` title
  - some text (To improve performance... Avo caches each record on the index view. More on that [here](./customization.html#cache-resources-on-the-index-view). There is a caveat with associated records/tables that the associated records are not updated when some associations are updated....   There are two things that you can do to prevent that
  - `###` subtitle (Option 1: Use `touch: true...)